sys/include/net/gnrc/sixlowpan/frag/rb.h

```diff
@@ -122,13 +185,13 @@ void gnrc_sixlowpan_frag_rbuf_remove(gnrc_sixlowpan_rbuf_t *rbuf);
  *                  Used to construct the @ref gnrc_netif_hdr_t of the completed
  *                  datagram. Must not be NULL.
  */
-void gnrc_sixlowpan_frag_rbuf_dispatch_when_complete(gnrc_sixlowpan_rbuf_t *rbuf,
-                                                     gnrc_netif_hdr_t *netif);
+void gnrc_sixlowpan_frag_rb_dispatch_when_complete(gnrc_sixlowpan_frag_rb_t *rbuf,
+                                                   gnrc_netif_hdr_t *netif);
 #else
-/* NOPs to be used with gnrc_sixlowpan_iphc if gnrc_sixlowpan_frag is not
+/* NOPs to be used with gnrc_sixlowpan_iphc if gnrc_sixlowpan_frag_rb is not
  * compiled in */
-#define gnrc_sixlowpan_frag_rbuf_remove(rbuf)       (void)(rbuf)
-#define gnrc_sixlowpan_frag_rbuf_dispatch_when_complete(rbuf, netif) \
+#define gnrc_sixlowpan_frag_rb_remove(rbuf)     (void)(rbuf)
+#define gnrc_sixlowpan_frag_rb_dispatch_when_complete(rbuf, netif) \
     (void)(rbuf); (void)(netif)
```

Comment:
I'm not suggesting to change this here as this is just about moving functionality around and that style is used throughout the rest of the module, but generally an empty `static inline` function will achieve the same result with less way for something to go wrong.

Reply:
Agreed, but let's do this later.